lib/translation_maps/marc_languages.properties

```diff
@@ -300,19 +318,21 @@ men = Mende
 mga = Irish, Middle (ca. 1100-1550)
 mic = Micmac
 min = Minangkabau
-#mis = Miscellaneous languages
+mis = Miscellaneous languages
 mkh = Mon-Khmer (Other)
+# mla is deprecated
 mla = Malagasy
 mlg = Malagasy
 mlt = Maltese
 mnc = Manchu
 mni = Manipuri
 mno = Manobo languages
 moh = Mohawk
+# mol is deprecated
 mol = Moldavian
 mon = Mongolian
-mos = Moore
-#mul = Multiple languages
+mos = Mooré
+mul = Multiple languages
```

Comment:
There are some tests that suggest we explicitly wanted to hide `mul` / `Multiple languages` (and `mis`, presumably). Have we decided to re-enable it, or was this a mistake?